.gitignore

```diff
@@ -39,3 +39,5 @@ clover.xml
 ###> custom ###
 .stylelintcache
 ###< custom ###
+
```

Comment:
Please revert this change.
Specific files/dirs to ignore should be placed in a global .gitignore file, not a project one.